static/lang/re-en.json

```diff
@@ -3154,6 +3154,9 @@
                     "PouncingWhirlUsedLabel": "Chimera Flail - Pouncing Whirl Used",
                     "StrideHalfLabel": "Chimera Flail - Stride at least half your Speed"
                 },
```

Comment:
Might not need the explicit label? If you do, should probably prefix the label with “Clandestine Cloak - “.